docs/pages/access-controls/access-graph/self-hosted.mdx

```diff
@@ -29,8 +29,19 @@ to Teleport Enterprise customers.
 - A TLS certificate for the Access Graph service
   - The TLS certificate must be issued for "server authentication" key usage,
     and must list the IP or DNS name of the TAG service in an X.509 v3 `subjectAltName` extension.
+  - Starting from version 1.20.4 of the Access Graph service, the container by default runs as a non-root user.
+    Make sure the certificate files are readable by the user running the container. You can set correct permissions with the following command:
+    ```console
+    $ sudo chown 65532 /path/to/tls.key
```

Comment:
Nit: we use `/etc/access_graph/tls.key` as the path in the example config further below, so maybe use that here as well.

Reply:
Changed